.env

```diff
@@ -1 +1,2 @@
 NEXT_PUBLIC_GLEAN_APP_BASE_URL=https://glean.io
+VERCEL_FORCE_NO_BUILD_CACHE=1
```

Comment:
Worth a comment here, like `# Had issues with new nav items not appearing without this`